models/chainlink/optimism/chainlink_optimism_schema.yml

```diff
@@ -133,6 +133,78 @@ models:
       - *proxy_address
       - *aggregator_address
 
+  - name: chainlink_optimism_ccip_onramps
+    meta:
+      blockchain: optimism
+      sector: CCIP
+      project: chainlink
+      contributors: synthquest
+    config:
+      tags: ['optimism','ccip','chainlink']
+    description: >
+        Chainlink CCIP messages sent from optimism
+    tests:
+      - dbt_utils.unique_combination_of_columns:
+          combination_of_columns:
+            - evt_index
+            - evt_tx_hash
+            - message
+    columns:
+      - &contract_address
+        name: contract_address
+        description: "Optimism CCIP ramp contract address"
+      - &evt_block_number
+        name: evt_block_number
+        description: "Block number which processed the unique transaction hash"
+      - &evt_block_time
+        name: evt_block_time
```

Comment:
There may be some duplicated `&` columns defined in the schema.yml files after merge with the main branch.

We may need to check and remove the duplicated & columns defined manually.

Reply:
These two lines may have the same definition:

https://github.com/duneanalytics/spellbook/blob/39bf886a61d1fd47c5c5f5c3eb49aa86a49cebd3/models/chainlink/optimism/chainlink_optimism_schema.yml#L153

https://github.com/duneanalytics/spellbook/blob/39bf886a61d1fd47c5c5f5c3eb49aa86a49cebd3/models/chainlink/optimism/chainlink_optimism_schema.yml#L229